requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
Can you please make sure our code is compatible with 3.8? this is the current version we have on farm.